app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -137,7 +137,8 @@ def pii_requested_but_locked?
     end
 
     def track_events
-      analytics.track_event(Analytics::SP_REDIRECT_INITIATED)
+      ial = sp_session[:ial2] ? 2 : 1
+      analytics.track_event(Analytics::SP_REDIRECT_INITIATED, ial: ial)
```

Comment:
can we just load the entire ial level? ex `ial2?strict=true`? This leaves out a lot of info

Reply:
We need this to match our billing reports also based on sp_session[:ial2]